java/ChangeLog

```diff
@@ -1,3 +1,9 @@
+02/10/2024 Moritz Vieli <moritz.vieli@gmail.com>
+ * v0.0.23
```

Comment:
I think you mean v0.0.3 in your versioning, but as mentioned this should be >0.1.0, I believe 0.2.0.

Reply:
Fixed with commit 79630929f9cde69e0dbff67f7e1cbdfa4e3acbb6